panoply/errors/exceptions.py

```diff
@@ -25,3 +25,9 @@ def __init__(self, message, code, exception_cls,
         self.database_id = database_id
         self.exception_cls = exception_cls
         self.created_at = datetime.utcnow()
+
+
+class TokenValidationException(PanoplyException):
```

Comment:
Please, import `TokenValidationException` in `panoply-python-sdk/panoply/errors/__init__.py`